Doc/library/sqlite3.rst

```diff
@@ -369,7 +369,7 @@ Module functions
    Create a Binary object to handle binary data in SQLite.
 
    :param data:
-       The binary data to be encapsulated. This should be a 
+      The binary data to be encapsulated. This should be a 
    :term:`bytes-like object`.
```

Comment:
```suggestion
      The binary data to be encapsulated. This should be a
      :term:`bytes-like object`.
```

Reply:
The lint error is here by the way.

Reply:
(And indents, it's really hard for me to do it with GH in comments sometimes because I don't see my own whitespaces).

To fix the lint:

- C/C previous usage (above in the file)
- Change the content.